Hunter/Assets/Scripts/Model/Entities/Animal.cs

```diff
@@ -36,5 +36,21 @@ public Animal()
             WanderAngle = random.Next(-1, 2);
             Velocity = new Vector2(xPos, yPos);
         }
+
+        public void GetEntititesInArea(List<Entity> allEntities)
+        {
+            List<Entity> areaEntities = new List<Entity>();
+
+            foreach (Entity entity in allEntities)
+            {
+                if (CollisionDetection.AreColliding(this, entity,
+                    BodySeekRadius, entity.BodyRadius) && this != entity)
```

Comment:
probably "this != entity" can be replaced by smth better (like Object.ReferenceEquals(this, entity))